### DISCUSSION TITLE
Tensorflow AI to automaticallly delete comments

### DISCUSSION THREAD

**Discussion by reesericci** [ThioJoe/YT-Spammer-Purge#50] [Ideas]
It would be really cool if this used something like TensorFlow AI to automatically monitor the comments and delete them.

**Comment by TechStudent10** [comment]
Yeah, and maybe automatically delete spam comments when they get commented.

Edit: Just realized that is exactly what you said,

**Reply by reesericci** [reply]
lol, we've all done it.

**Comment by TechStudent10** [comment]
But I don't think you can monitor YouTube videos with stuff like webhooks. So you would have to keep refreshing the current script I guess.

**Reply by reesericci** [reply]
yeah, just ping that api endpoint until they say no amirite?

**Comment by http403** [comment]
I think it is more appropriate to change the title to be more descriptive of the idea.

**Reply by reesericci** [reply]
I agree now that it is discussions and is in the "ideas" category.

**Comment by ThioJoe** [comment]
It would be cool but would also be wildly outside the purpose of the program for the foreseeable future.

**Reply by reesericci** [reply]
That's fair, but I thought it would be an interesting concept to discuss.

**Comment by seraphya** [comment]
While looking for training data I found a project that was alrady doing this. 

https://github.com/Savjee/yt-spam-classifier
